devel/libcbor/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libcbor
+license             MIT License
+version             0.8.0
```

Comment:
once you use github PortGroup, it will add this line for you and you won't need to put it here again